include/seqan3/alphabet/cigar/cigar.hpp

```diff
@@ -159,16 +159,22 @@ class cigar : public alphabet_tuple_base<cigar, uint32_t, exposition_only::cigar
     /*!\name Write functions
      * \{
      */
-    /*!\brief Assign from the string representation.
+    /*!\brief Assign from a std::string_view.
      * \details
-     * \experimentalapi{Experimental since version 3.1.}
+     *
+     * In order to avoid unnecessary copies, you can initialise a seqan3::cigar from a std::string_view that points to a
+     * contigous char array that stores the cigar string.
```

Comment:
```suggestion
     * In order to avoid unnecessary copies, you can initialise a seqan3::cigar from a std::string_view that contains
     * the cigar string.
```